Upstream-Status: Pending

This patch fixes following issue with automake 1.12

| automake: warnings are treated as errors
| /srv/home/nitin/builds/build-gcc47/tmp/sysroots/x86_64-linux/usr/share/automake-1.12/am/ltlibrary.am: warning: 'libcompat.la': linking libtool libraries using a non-POSIX
| /srv/home/nitin/builds/build-gcc47/tmp/sysroots/x86_64-linux/usr/share/automake-1.12/am/ltlibrary.am: archiver requires 'AM_PROG_AR' in 'configure.ac'

Signed-Off-By: Nitin A Kamble <nitin.a.kamble@intel.com>
2012/05/03

Index: check-0.9.8/configure.ac
===================================================================
--- check-0.9.8.orig/configure.ac
+++ check-0.9.8/configure.ac
@@ -103,6 +103,10 @@ AC_PROG_INSTALL
 AC_PROG_LN_S
 AC_PROG_LIBTOOL
 
+# automake 1.12 seems to require this, but automake 1.11 doesn't recognize it
+m4_pattern_allow([AM_PROG_AR])
+AM_PROG_AR
+
 # add these options to CFLAGS if the compiler supports them
 AC_DEFUN([AX_CFLAGS_ADD],[AX_C_CHECK_FLAG($1, , , CFLAGS="$CFLAGS $1")])
 AX_CFLAGS_WARN_ALL_ANSI
